### PR TITLE
chore(css): Move CSS examples - Logical properties, multicol, overflow, scroll snap

### PR DESCRIPTION
**Pull request by bsmth**
### Description

As part of an initiative to reduce repo maintenance & code duplication, this PR moves some external examples into content. This PR converts the following macros to live samples:

```
{{EmbedGHLiveSample("css-examples/logical/intro-grid-example.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/logical/float.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/logical/positioning-inset.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/logical/text-align.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/logical/margin-longhands.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/logical/padding-longhands.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/logical/border-longhands.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/logical/size-inline-block.html", '100%', 500)}}
{{EmbedGHLiveSample("css-examples/logical/size-min.html", "100%", 500)}}
{{EmbedGHLiveSample("css-examples/logical/size-max.html", "100%", 500)}}
{{EmbedGHLiveSample("css-examples/logical/size-resize.html", "100%", 700)}}
{{EmbedGHLiveSample("css-examples/multicol/basics/column-count.html", '100%', 550)}}
{{EmbedGHLiveSample("css-examples/multicol/basics/column-width.html", '100%', 550)}}
{{EmbedGHLiveSample("css-examples/multicol/basics/column-count-width.html", '100%', 550)}}
{{EmbedGHLiveSample("css-examples/multicol/fragmentation/break-inside.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/multicol/fragmentation/break-before.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/multicol/fragmentation/orphans.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/multicol/overflow/image.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/multicol/overflow/image-max-width.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/multicol/overflow/overflow-inline.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/multicol/overflow/min-height.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/modules/multicol.html", '100%', 650)}}
{{EmbedGHLiveSample("css-examples/multicol/spanning/h2-span.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/multicol/spanning/nested-h2-span.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/multicol/spanning/mpb-span.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/multicol/balancing/balance.html", '100%', 550)}}
{{EmbedGHLiveSample("css-examples/multicol/balancing/auto.html", '100%', 550)}}
{{EmbedGHLiveSample("css-examples/multicol/styling/column-gap.html", '100%', 750)}}
{{EmbedGHLiveSample("css-examples/multicol/styling/column-rule.html", '100%', 550)}}
{{EmbedGHLiveSample("css-examples/multicol/styling/column-rule-wide.html", '100%', 550)}}
{{EmbedGHLiveSample("css-examples/modules/overflow.html", '100%', 320)}}
{{EmbedGHLiveSample("css-examples/scroll-snap/mandatory-y.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/scroll-snap/mandatory-proximity.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/scroll-snap/align.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/scroll-snap/scroll-padding.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/scroll-snap/scroll-padding-sticky.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/scroll-snap/scroll-margin.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/modules/scroll_snap.html", '100%', 250)}}
```

### Motivation

Motivation is described in the project issue.

### Related issues and pull requests

- https://github.com/mdn/mdn/issues/597

## TODO

- [x] update files/en-us/web/css/css_scroll_snap/index.md with latest from `main`
- [x] update files/en-us/web/css/css_scroll_snap/basic_concepts/index.md with latest from `main`
- [x] Remove "The HTML is identical to".. based on reviewer feedback in related PR.

### CODE REVIEW

File: files/en-us/web/css/css_logical_properties_and_values/floating_and_positioning/index.md [hunk 1]
```diff
@@ -39,7 +39,57 @@ The physical values used with the {{cssxref("float")}} and {{cssxref("clear")}}
 
 This example has two boxes — the first has the box floated with `float: left`, the second with `float: inline-start`. If you change the `writing-mode` to `vertical-rl` or the `direction` to `rtl` you will see that the left-floated box always sticks to the left, whereas the `inline-start`-floated item follows the `direction` and `writing-mode`.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/css_logical_properties_and_values/floating_and_positioning/index.md:40]: The instructions here are a little bit confusing — changing the `writing-mode` to `vertical-rl` doesn't have any noticeably different effect on the two floated boxes. It is really `direction` that has an effect here. Maybe change the instructions to tell the reader to add `direction: rtl` to the `.inner` ruleset and observe the effect.

**Review by chrisdavidmills — APPROVED** [review]
@bsmth more great work here; nice one. Mostly these look fine; I just had a small comment on one bit of text. I'm not convinced it is that important really, so approving.

File: files/en-us/web/css/css_logical_properties_and_values/floating_and_positioning/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/web/css/css_logical_properties_and_values/floating_and_positioning/index.md:40]: True, I see what you mean, the prose doesn't match the example. Your suggestion is nice, thank you!

Comment (bsmth) [reply files/en-us/web/css/css_logical_properties_and_values/floating_and_positioning/index.md]: Added something in https://github.com/mdn/content/pull/36730/commits/d99b08eeec4b1f1645f6d7b2ace2b5ee332b1e91 which I think is a little better, what do you think?

File: files/en-us/web/css/css_logical_properties_and_values/floating_and_positioning/index.md [hunk 2]
```diff
@@ -37,9 +37,62 @@ In addition to these mapped properties, there are some additional shorthand prop
 
 The physical values used with the {{cssxref("float")}} and {{cssxref("clear")}} properties are `left`, `right` and `both`. The CSS logical properties and values module defines the values `inline-start` and `inline-end` as mappings for `left` and `right`.
 
-This example has two boxes — the first has the box floated with `float: left`, the second with `float: inline-start`. If you change the `writing-mode` to `vertical-rl` or the `direction` to `rtl` you will see that the left-floated box always sticks to the left, whereas the `inline-start`-floated item follows the `direction` and `writing-mode`.
-
-{{EmbedGHLiveSample("css-examples/logical/float.html", '100%', 700)}}
+In the example below, the first box is floated with `float: left`, and the second with `float: inline-start`.
+If you apply `direction: rtl` to the `.inner` element, the left-floated box always stays on the left, whereas the `inline-start`-floated item follows the `direction` of the text.
+You can also try `writing-mode: vertical-rl` to see the difference this makes to the direction of block layout in combination with text `direction` values:
```
Comment (bsmth) [inline files/en-us/web/css/css_logical_properties_and_values/floating_and_positioning/index.md:42]:
```suggestion
If you apply `direction: rtl` to the `.inner` selector, the left-floated box always stays on the left, whereas the `inline-start`-floated item follows the `direction` of the text.
You can combine this with `writing-mode: vertical-rl` to see the difference of block layout in combination with `direction` values.
```

File: files/en-us/web/css/css_logical_properties_and_values/floating_and_positioning/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/web/css/css_logical_properties_and_values/floating_and_positioning/index.md]: Thanks, Chris. I'm going to merge this one now. If you've other comments I can also improve in follow-up PRs.

**Pull request merged**